src/main/java/org/reaktivity/nukleus/http2/internal/NukleusWriteScheduler.java

```diff
@@ -16,96 +16,124 @@
 package org.reaktivity.nukleus.http2.internal;
 
 import org.agrona.MutableDirectBuffer;
+import org.agrona.concurrent.UnsafeBuffer;
+import org.reaktivity.nukleus.buffer.MemoryManager;
 import org.reaktivity.nukleus.function.MessageConsumer;
 import org.reaktivity.nukleus.http2.internal.types.Flyweight;
-import org.reaktivity.nukleus.http2.internal.types.stream.DataFW;
+import org.reaktivity.nukleus.http2.internal.types.stream.TransferFW;
 
-class NukleusWriteScheduler
+import java.io.Closeable;
+
+class NukleusWriteScheduler implements Closeable
 {
+    private static final int REGION_SIZE = 65536;
+
```

Comment:
Please add to `Htto2Configuration` following approach similar to `TcpConfiguration.transferCapacity()` and `TlsConfiguration.transferCapacity()`.